apps/frontend/src/libs/helpers/debounce/libs/constants.ts

```diff
@@ -0,0 +1,3 @@
+const DEBOUNCE_TIMEOUT = 300;
```

Comment:
we do not need it globally. it should be more specific and put it near the place where you will use it